app/services/model_exporters/team_exporter.rb

```diff
@@ -57,7 +54,9 @@ def team(team)
         custom_fields: team.custom_fields,
         repositories: team.repositories.map { |r| repository(r) },
         tiny_mce_assets: team.tiny_mce_assets,
-        protocols: team.protocols.where(my_module: nil).map { |pr| protocol(pr) },
+        protocols: team.protocols.where(my_module: nil).map do |pr|
+          protocol(pr)
+        end,
```

Comment:
I think this change is not needed. One liner is better.

Reply:
than it will not fit into the line